test/python/primitives/test_backend_sampler.py

```diff
@@ -193,8 +193,8 @@ def test_run_2qubit(self, backend):
 
         self.assertDictAlmostEqual(result.quasi_dists[0], {0: 1}, 0.1)
         self.assertDictAlmostEqual(result.quasi_dists[1], {1: 1}, 0.1)
-        self.assertDictAlmostEqual(result.quasi_dists[2], {2: 1}, 0.1)
-        self.assertDictAlmostEqual(result.quasi_dists[3], {3: 1}, 0.1)
+        self.assertDictAlmostEqual(result.quasi_dists[2], {2: 1}, 0.2)
```

Comment:
Why do you need to change delta values?

Reply:
This change implies that this PR amplifies the error of `BackendSampler` with a noise model.
Does this PR set a different noise parameter values?

Reply:
Strange.. I'll take a deeper look at where the increase of error comes from early next week.

Reply:
I've looked into the above unexpected behavior and found two issues:
- A bug in Aer where excessive errors (other than `ROError`) are added to `measure` instructions only when the noises are generated from `Target` object
- An issue in `convert_to_target` in Terra, that drops asymmetric readout error properties (`prob_meas0_prep1` and `prob_meas1_prep0`) defined in `props_dict` since they are not defined in `InstructionProperties`. We may need to accept this or introduce a new `MeasureInstructionProperties` subclassing `InstructionProperties`.

I'll fix the first bug as soon as possible (hopefully the fixe should be included in aer 0.11.2).
As for the second issue, I'll create an issue to discuss how to address it.
Sorry, please wait for them, @cometta

Reply:
Thanks for looking into this, Itoko-san.

@cometta: This is why we have tests - when stuff starts failing unexpectedly, we need to figure out why it's failing, not modify the tests to mask it.  Tests are more than just things that need to be made to pass, they're also a tool for us to help find errors in our code.